src/arrayMethodJoin.js

```diff
@@ -5,8 +5,27 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let arr = '';
+    let x = String(separator);
```

Comment:
```suggestion
    let joinedString = '';
```

Reply:
why you create variable with string - named arr?

Reply:
i thought it is almost the same as string named joinedString